### ISSUE TITLE
elf.transformation.elastix_parser cannot be imported

### ISSUE THREAD

**Issue by jhennies** [mobie/mobie-utils-python#13]
Hi @constantinpape,

After installing mobie-utils-python by 
```
$ git clone https://github.com/mobie/mobie-utils-python
$ cd mobie-utils-python
$ conda env create -f environment.yaml
$ conda activate mobie
$ pip install -e .
```

when running
`from mobie.initialization import make_dataset_folder`

I get 
```
Traceback (most recent call last):
  File "<frozen importlib._bootstrap>", line 983, in _find_and_load
  File "<frozen importlib._bootstrap>", line 953, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/g/schwab/hennies/src/github/membranet_segmentation/tmp/mobie-utils-python/mobie/__init__.py", line 1, in <module>
    from .image_data import add_image_data
  File "/g/schwab/hennies/src/github/membranet_segmentation/tmp/mobie-utils-python/mobie/image_data.py", line 6, in <module>
    from mobie.import_data import import_raw_volume
  File "/g/schwab/hennies/src/github/membranet_segmentation/tmp/mobie-utils-python/mobie/import_data/__init__.py", line 4, in <module>
    from .registration import apply_registration
  File "/g/schwab/hennies/src/github/membranet_segmentation/tmp/mobie-utils-python/mobie/import_data/registration/__init__.py", line 1, in <module>
    from .apply_registration import apply_registration
  File "/g/schwab/hennies/src/github/membranet_segmentation/tmp/mobie-utils-python/mobie/import_data/registration/apply_registration.py", line 10, in <module>
    from elf.transformation import elastix_parser
ImportError: cannot import name 'elastix_parser' from 'elf.transformation' (/g/schwab/hennies/miniconda3/envs/mobie/lib/python3.7/site-packages/elf/transformation/__init__.py)
```

From looking into elf and the releases, it seems you have the elastix functionality not in the newest releases.

I guess, I don't need the functionality, so commenting out the import of this and other elastix related imports would be a workaround for me.

However it might make sense to put `try except` statements around the imports here or generate and integrate a newer version of elf into the environment yaml. 

Cheers,
Julian

**Comment by constantinpape** [comment]
> However it might make sense to put `try except` statements around the imports here or generate and integrate a newer version of elf into the environment yaml.

Yeah, I agree it would be best to wrap this in a `try ImportError except`.

**Comment by constantinpape** [comment]
@jhennies this should be fixed by b6f22f3. Could you check if it works for you now?

**Comment by jhennies** [comment]
Thanks @constantinpape, it works perfectly
